Cart.Business/Implementations/CartingService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Cart.Business.Interfaces;
+using Cart.Business.Mappers;
+using Cart.Business.Models;
+using Cart.DataAccess.Interfaces;
+
+namespace Cart.Business.Implementations
+{
+    public class CartingService : ICartingService
```

Comment:
1) It would be nice to add `sealed` keyword to classes that do not have child classes. You could get 2 benefits:
 - CLR will see that class cannot have child classes, therefore `call` instruction be used instead of `callvirt`. Small performance benefit :)
 - If you need to inherit the service , it would be simpler to remove sealed keyword (extend the functionality is simpler than restrict it)

```suggestion
    public sealed class CartingService : ICartingService
```